### PR TITLE
add database support of pilotage

### PR DESCRIPTION
**Pull request by wangkirin**
Signed-off-by: Wang Qilin <qilin.wang@huawei.com>

### CODE REVIEW

File: pilotage/model/stage.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	Name       string
+	Title      string
+	Sequencing string
+	CreateAt   time.Time
+	UpdateAt   time.Time
+}
+
+type StageDataV1 struct {
+	ID      string
+	StageID string
```
Comment (wangkirin) [inline pilotage/model/stage.go]: @genedna  I don't really understand why it should be FlowDataID here ?  I think everytime the stage runs ,it depends on data

**Comment by genedna** [comment]
@wangkirin Would I review now?

**Comment by wangkirin** [comment]
@genedna  I have already finish the work, you can start to review now

**Comment by genedna** [comment]
LGTM

**Pull request merged**